### ISSUE TITLE
Doesn't show errors in status bar, in the gutter or in the code

### ISSUE THREAD

**Issue by GoOz** [kungfusheep/SublimeLinter-contrib-stylelint#27]
Can't decide if it's a SublimeLinter's bug or a Stylelint linter's bug but since the scss linter works just fine, i came to you.

So basically, the linter finds the .stylelintrc file and lint just fine but can't seem to show me any error anywhere on the UI. Errors only appears in the console when i enable the debug mode of SublimeLinter.

Apparently it's loading just fine too. Any idea of what could make a mess ?

**Comment by dostrog** [comment]
The same problem (Sublime Text build 3103)

**Comment by borisirota** [comment]
Same here (Sublime Text build 3103) only I can't see errors in the console in debug mode.
When running `stylelint` from the cli (installed locally), everything works fine.

**Comment by ivan-ravecca** [comment]
Unfortunately, same for me (on the same build)

**Comment by Cap32** [comment]
The same problem at `stylelint v6.x`.

`v5.4.0` resolved the problem.

**Comment by arianjinx** [comment]
@Cap32 already downgrade it to `5.4.0` and still no luck.

Edited: oops it's work, sorry haha. But linter broke on unclosed block.

**Comment by wroughtec** [comment]
Think it could be to do with the recent update to Stylelint to version 6 as it in complaining in the console or me:
Unknown word   CssSyntaxError

In v6 in the change log:
Changed: CssSyntaxError is no longer thrown but reported alongside warnings.

So perhaps this has broken something in Sublime's implementation. Was working fine in version 5.4 for me

Have mentioned in the stylelint as well as it maybe a simple change to fix:
https://github.com/stylelint/stylelint/issues/1146

**Comment by wroughtec** [comment]
Hmm Looks like that error was not liking Sass variables and was masking all other errors in the console

**Comment by steffenweber** [comment]
I had the same problem with errors showing up in the console but not in the UI. Manually applying the change in commit https://github.com/kungfusheep/SublimeLinter-contrib-stylelint/commit/3ab856778df9539c7993abec00b6e76958270fc3 (which does not seem to be released yet) has fixed the problem for me.

**Comment by arianjinx** [comment]
Big thanks @steffenweber now it works completely in these version.

```
"stylelint": "^6.2.2",
"stylelint-config-standard": "^6.0.0"
```

and modify `linter.py` like you instructed.

**Comment by GoOz** [comment]
YES ! Thanks @steffenweber ! Works for me too. 
That will do the trick until the plugin gets updated. :-)

**Comment by wroughtec** [comment]
Unfortunately this does not fix everything:

``` css
.class--discount {
  &::before {
    @extend %#{$fa-css-prefix}-check;
  }
}
```

gives the following error:
Unknown word CssSyntaxError

It does not like the SASS variable as if I remove them it works fine.

Which then blocks all other errors/warnings on the page :(

Any suggestions for this

**Comment by wroughtec** [comment]
Ah this appears to be not a sublime plugin error as I can reproduce in CLI will move this over to them

**Comment by wroughtec** [comment]
Ok when I run in CLI:
stylelint --syntax scss ./scss/_components/_c-main-nav.scss

I don't get this error is this an option we can pass for Sublimtext?

**Comment by steffenweber** [comment]
That seems to be issue #15.

**Comment by kevinSuttle** [comment]
Still broken in Build 3114. Any ETA in sight? Looks like https://github.com/kungfusheep/SublimeLinter-contrib-stylelint/commit/3ab856778df9539c7993abec00b6e76958270fc3 did _not_ fix it.
